recipes/phynder/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -x
+set +e
+
+export CFLAGS="$CFLAGS -I$PREFIX/include"
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
+export CPATH=${PREFIX}/include
+
+## Static lib required by phynder, thus following instructions
+## rather than trying to take from a conda htslib
+git clone https://github.com/samtools/htslib.git
```

Comment:
I wasn't sure what to do in this case. I thought about it, but the developer doesn't specify any version restrictions.

Given in the dependency pinning in the meta.yaml is preferred to be not strict (although of course for other reasons) I thought maybe worth keeping with the same philosophy here, unless you think it would be safer?